src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1589,26 +1570,10 @@ int computeIndexingBin() {
             // then treat this as a one base alignment for indexing purposes.
             alignmentEnd = alignmentStart + 1;
         }
-        return GenomicIndexUtil.regionToBin(alignmentStart, alignmentEnd);
-    }
 
-
-    /**
-     * Gets indexing bin if it is present otherwise compute it.
-     * @param alignment to compute indexing bin for.
-     * @return indexing bin.
-     */
-    public int computeIndexingBinIfAbsent(final SAMRecord alignment) {
```

Comment:
change in public API...perhaps deprecate first?

Reply:
Done.  Its such a horrible method and should never have been public in the first place, but just in case.